en/csql.rst

```diff
@@ -221,38 +225,64 @@ To display the option list in the prompt, execute the **csql** utilities without
     
     ::
     
-        $ csql testdb@localhost -c "select * from test_tbl" -t
- 
-        col1 col2 col3
-        string1 12:16:10.090 PM 10/23/2014
-        string2 12:16:10.090 PM 10/23/2014
-        string3 12:16:10.090 PM 10/23/2014
-        string4 12:16:10.090 PM 10/23/2014
-        string5 12:16:10.090 PM 10/23/2014
-        string6 12:16:10.090 PM 10/23/2014
-        string7 12:16:10.090 PM 10/23/2014
-        string8 12:16:10.090 PM 10/23/2014
-        string9 12:16:10.090 PM 10/23/2014
-        string10 12:16:10.090 PM 10/23/2014
+        $ csql demodb -c "select * from athlete where code between 12762 and 12765" -t
+
+        code    name    gender  nation_code event
+        12762   O'Brien Dan M   USA Athletics
+        12763   O'Brien Leah    W   USA Softball
+        12764   O'Brien Shaun William   M   AUS Cycling
+        12765   O'Brien-Amico Leah  W   USA Softball
  
+.. option:: -q, --query-output
+
+    This option displays the result for easy use in insert queries, only show column names and values and works with **-c** or **-i** option. Each column name and value is separated by a comma or a single character of the **--delimiter** option and all results except for numeric types is enclosed by a single quote or a single character of the **--enclosure** option. If the enclosure is single quote, the single quote in the results is replaced with two ones. It is ignored when it is given with **-l** option.
+
+    ::
+
+        $ csql demodb -c "select * from athlete where code between 12762 and 12765" -q
+
+        code,name,gender,nation_code,event
+        12762,'O''Brien Dan','M','USA','Athletics'
+        12763,'O''Brien Leah','W','USA','Softball'
+        12764,'O''Brien Shaun William','M','AUS','Cycling'
+        12765,'O''Brien-Amico Leah','W','USA','Softball'
+
+    ::
+
+        $ csql demodb -c "select * from athlete where code between 12762 and 12765" -q --delimiter="" --enclosure="\""
+
+        code,name,gender,nation_code,event
+        12762,"O'Brien Dan","M","USA","Athletics"
+        12763,"O'Brien Leah","W","USA","Softball"
+        12764,"O'Brien Shaun William","M","AUS","Cycling"
+        12765,"O'Brien-Amico Leah","W","USA","Softball"
+
+.. option:: -d, --loaddb-output
+
+    This option displays the result for easy use in loaddb utility, only show column names and values and works with **-c** or **-i** option. Each column name and value is separated by a space and all results except for numeric types is enclosed by a single quote. The single quote in the results is replaced with two ones and the result of the enum type is outputed by an index instead of a value. This opton is ignored when it is given with **-l** option.
```

Comment:
1. only show column names and values and works with **-c** or **-i** option. --> only show column names and values, and works with **-c** or **-i** option. (comma added)

2. Each column name and value is separated by a space and all results except for numeric types is enclosed by a single quote.

- Each column name and value is... --> Each column name and value are... (is --> are)

- The sentence is too long so maybe adding ';' before 'and all results except for numeric types...'

- all results except for numeric types is enclosed by ... --> all results except for numeric types are enclosed by ( is --> are)

3. the result of the enum type is outputed --> the result of the enum type is outputted (past perfect tense of output is outputted)

Reply:
thank you for reviewing, i will apply your comment.